### PR TITLE
Mapping the content priority from 0 - 1000 to be between 0 - 1.

### PR DESCRIPTION
**Pull request by MartinsIrbe**

### CODE REVIEW

File: src/main/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractor.java [hunk 1]
```diff
@@ -205,18 +205,17 @@ public void init() {
     public void setDescribedFieldsFrom(BtVodEntry row, Described described) {
         described.setDescription(row.getDescription());
         described.setLongDescription(row.getProductLongDescription());
-        if (row.getProductPriority() != null && Double.valueOf(row.getProductPriority()) > 0) {
+        if (row.getProductPriority() != null && Double.valueOf(row.getProductPriority()) >= 0) {
```
Comment (Spartakovic) [inline src/main/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractor.java]: can it be less then zero?

Comment (MartinsIrbe) [inline src/main/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractor.java]: @Spartakovic well I think so as previous logic had this check. :)

File: src/test/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractorTest.java [hunk 2]
```diff
@@ -237,4 +243,56 @@ public void testGetAliasesIfSynthesisedFromEntry() throws Exception {
                 is(BT_VOD_SYNTHESISED_FROM_ID_ALIAS_NAMESPACE));
         assertThat(Iterables.get(aliases, 1).getValue(), is(entry.getId()));
     }
+
+    @Test
+    public void whenPriorityIsNotPresentThenSetToBe1ByDefault() {
```
Comment (Dalamar42) [inline src/test/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractorTest.java]: Add another test for negative values.

File: src/main/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractor.java [hunk 3]
```diff
@@ -203,20 +203,35 @@ public void init() {
     }
     
     public void setDescribedFieldsFrom(BtVodEntry row, Described described) {
+        // The old range is between 1000 to 1
+        Double oldMaxRange = 1D;
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractor.java]: Set these values as constants. Also naming could be more descriptive, e.g. `BT_MAX_PRIORITY`, `MBST_MAX_PRIORITY`

File: src/main/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractor.java [hunk 4]
```diff
@@ -203,20 +203,35 @@ public void init() {
...
         if (row.getProductPriority() != null && Double.valueOf(row.getProductPriority()) > 0) {
-            Double priority = Double.valueOf(row.getProductPriority());
-            if (priority > 0) {
-                described.setPriority(new Priority(Math.abs(priority) * 0.33,
-                        new PriorityScoreReasons(
-                                ImmutableList.of(""),
-                                ImmutableList.of("")
-                        )
-                ));
+            Double oldScore = Double.valueOf(row.getProductPriority());
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractor.java]: Naming: `btPriority`

File: src/main/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractor.java [hunk 5]
```diff
@@ -61,6 +62,10 @@
     private final Topic tvBoxsetTopic;
     private final Topic subCatchupTopic;
 
+    private final Double BT_MAX_PRIORITY = 1D;
+    private final Double BT_MIN_PRIORITY = 1000D;
+    private final Double MBST_MAX_PRIORITY = 1D;
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/btvod/BtVodDescribedFieldsExtractor.java]: Add MBST_MIN_PRIORITY = 0D; here as well otherwise it's not obvious that's what it is
